tokenize_rt.py

```diff
@@ -82,29 +112,31 @@ def src_to_tokens(src):
     return tokens
 
 
-def tokens_to_src(tokens):
+def tokens_to_src(tokens):  # type: (Iterable[Token]) -> Text
     return ''.join(tok.src for tok in tokens)
 
 
 def reversed_enumerate(tokens):
+    # type: (Sequence[Token]) -> Iterator[Tuple[int, Token]]
     for i in reversed(range(len(tokens))):
         yield i, tokens[i]
 
 
-def parse_string_literal(src):
+def parse_string_literal(src):  # type: (Text) -> Tuple[Text, Text]
     """parse a string literal's source into (prefix, string)"""
     match = _string_re.match(src)
-    return match.group(1), match.group(2)
+    # ignore the no-match case to preserve pre-typed behavior
+    return match.group(1), match.group(2)  # type: ignore
```

Comment:
instead of `# type: ignore`, `assert match is not None` instead

Reply:
I thought about that but that's going to be a breaking change (`AttributeError` -> `AssertionError`). Is that OK?

Reply:
📟

Reply:
yes